ocs_ci/deployment/cloud.py

```diff
@@ -119,3 +120,4 @@ def deploy(self, log_cli_level="DEBUG"):
                     logger.error(ex)
             raise e
         self.test_cluster()
+        label_nodes_region()
```

Comment:
is this needed for other cloud platforms like aws, gcp ?

Reply:
This is IBM Cloud IPI specific because of we had issue to use COS as noobaa backing store.  @clacroix12 is this correct way to have it in the base IPI deployment class? Should we for now to only add this label for IBM Cloud IPI where we only see issue there?

Also the lable is only IBM Cloud specific, so it's not generic cloud specific.

Reply:
Moved this function call to the `post_ocp_deploy` method in the base deployment class as well as added a platform/deployment_type check.